packages/core/admin/admin/src/pages/UseCasePage/index.js

```diff
@@ -74,25 +73,25 @@ const UseCasePage = () => {
   const { formatMessage } = useIntl();
   const [role, setRole] = useState();
   const [otherRole, setOtherRole] = useState('');
+  const { post } = useFetchClient();
+
   const { firstname, email } = auth.getUserInfo();
   const { hasAdmin } = parse(location?.search, { ignoreQueryPrefix: true });
   const isOther = role === 'other';
 
-  const handleSubmit = (skipPersona) => {
+  const handleSubmit = async (event, skipPersona) => {
+    event.preventDefault();
     try {
-      axios({
-        method: 'POST',
-        url: 'https://analytics.strapi.io/register',
-        data: {
-          email,
-          username: firstname,
-          firstAdmin: Boolean(!hasAdmin),
-          persona: {
-            role: skipPersona ? undefined : role,
-            otherRole: skipPersona ? undefined : otherRole,
-          },
+      const dataToSend = {
+        email,
+        username: firstname,
+        firstAdmin: Boolean(!hasAdmin),
+        persona: {
+          role: skipPersona ? undefined : role,
+          otherRole: skipPersona ? undefined : otherRole,
```

Comment:
> sorry I can't see the difference

i had to do it in chunks because of how GH works: https://github.com/strapi/strapi/pull/16628#discussion_r1189495072